src/css/jwplayer/imports/jwplayerlayout.less

```diff
@@ -19,12 +19,12 @@
     }
 
     // Aspect Ratio styles
-    &.jw-flag-aspect-mode {
+    &.jw-flag-aspect-mode .jw-wrapper {
         /* Height auto required for displaying aspect ratio correctly */
         /* stylelint-disable-next-line declaration-no-important */
         height: auto !important;
```

Comment:
This change also removes height settings from the player container which could result in regressions.